test/behave/steps/wiki.py

```diff
@@ -1,11 +1,30 @@
 import time 
 
-from behave import then
-
+from behave import then, when
+from mycroft.messagebus import Message
 from mycroft.audio import wait_while_speaking
+from test.integrationtests.voight_kampff import then_wait
 
-@then('wait while speaking')
-def handle_wait(context):
-    # add sleep to ensure there is a delay when using a dummy TTS eg CI
+@then('dialog is stopped')
+def dialog_is_stopped(context):
     time.sleep(3)
-    wait_while_speaking()
\ No newline at end of file
+    context.bus.emit(Message('mycroft.audio.speech.stop',
+                             data={},
+                             context={}))
+    time.sleep(1)
+
+@then('"{skill}" should reply with dialog "{dialog}"')
+def then_dialog(context, skill, dialog):
```

Comment:
No I believe that is "with dialog from". I did not want to modify the original and since this one is different I stuck it here. I will create a feature request ticket to see if these can be reconciled somehow.